packages/component/src/Tracker.js

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import { useCallback, useEffect, useRef } from 'react';
+
+import { checkSupport as supportWorker } from './Utils/downscaleImageToDataURLUsingWorker';
+import { speechSynthesis } from './Speech/BypassSpeechSynthesisPonyfill';
+import useDebugDeps from './hooks/internal/useDebugDeps';
+import useLanguage from './hooks/useLanguage';
+import useTrackDimension from './hooks/useTrackDimension';
+import useWebSpeechPonyfill from './hooks/useWebSpeechPonyfill';
+
+function useEffectWithCounter(fn, deps) {
+  const counterRef = useRef(0);
+  const cachedFn = useCallback(fn, deps);
+
+  useEffect(() => cachedFn(counterRef.current++), [cachedFn, ...deps]);
+}
+
+const Tracker = () => {
+  const [language] = useLanguage();
+  const [webSpeechPonyfill] = useWebSpeechPonyfill();
+  const trackDimension = useTrackDimension();
+
+  // TODO: Track how many of them customized the following:
```

Comment:
Does 'them' mean the list below?
Should we file an issue here?

Reply:
Reminder about other TODO's below. Should they be filed, or are they meant to be addressed in this PR?